### DISCUSSION TITLE
Problem with install

### DISCUSSION THREAD

**Discussion by PlexxTron** [itsToggle/plex_debrid#637] [Q&A]
Hi, I hope someone can help me as I am going around in circles for hours.

I'm trying this command:
inside the terminal, enter the command pip install -r requirements.txt and press enter
but keep getting this error:
Building wheels for collected packages: regex
  Building wheel for regex (setup.py) ... error
  error: subprocess-exited-with-error

  × python setup.py bdist_wheel did not run successfully.
  │ exit code: 1
  ╰─> [13 lines of output]
      running bdist_wheel
      running build
      running build_py
      creating build
      creating build\lib.win-amd64-cpython-312
      creating build\lib.win-amd64-cpython-312\regex
      copying regex_3\__init__.py -> build\lib.win-amd64-cpython-312\regex
      copying regex_3\regex.py -> build\lib.win-amd64-cpython-312\regex
      copying regex_3\_regex_core.py -> build\lib.win-amd64-cpython-312\regex
      copying regex_3\test_regex.py -> build\lib.win-amd64-cpython-312\regex
      running build_ext
      building 'regex._regex' extension
      error: Microsoft Visual C++ 14.0 or greater is required. Get it with "Microsoft C++ Build Tools": https://visualstudio.microsoft.com/visual-cpp-build-tools/
      [end of output]

  note: This error originates from a subprocess, and is likely not a problem with pip.
  ERROR: Failed building wheel for regex
  Running setup.py clean for regex
Failed to build regex
ERROR: Could not build wheels for regex, which is required to install pyproject.toml-based projects

Can anyone help me?

Thanks

**Comment by cubbieblue16** [comment]
I am having the same issue here. Were you able to fix it?

**Reply by PlexxTron** [reply]
Hi there

Yes I solved it by installing an older version of python (don't know which). Hope that helps.

**Reply by Mamoon-B** [reply]
what a sucky response. you could have just checked which version of python you installed.

3.11.4 worked for me. thanks for pointing in the right direction i guess, although i had to try various versions and waste a ton of time in HOPES this was a solution.

**Reply by cubbieblue16** [reply]
You should be using 3.10. That worked fine.

**Reply by PlexxTron** [reply]
Hi there

I'm not in my home country at the moment so no access to PC to check version of python. I just wanted to give an answer that might help. Wasn't meant to be a sucky answer.

**Reply by Mamoon-B** [reply]
gotcha, no hard feelings. thank you for your contributions.
